organisaatio-ui/src/components/Sivut/Ryhmat/Ryhmat.tsx

```diff
@@ -52,51 +50,46 @@ const Ryhmat: React.FC = (props) => {
                     </a>
                 );
             },
-            collapse: true,
+            accessor: (values: any) => {
+                return values.nimi[language] || values.nimi.fi || values.nimi.sv || values.nimi.en || '';
+            },
+            //collapse: true,
         },
         {
             Header: i18n.translate('RYHMAN_TYYPPI'),
+            id: 'Tyyppi',
             Cell: ({ row }: any) => {
-                if (row.original.tyypit.length > 0) {
-                    return row.original.tyypit
-                        .map((tyyppi: string) => {
-                            const koodityyppi = ryhmaTyypitKoodisto
-                                .koodit()
-                                .find((rt) => rt.uri === tyyppi.slice(0, -2));
-                            if (koodityyppi) {
-                                return (
-                                    koodityyppi.nimi[language] ||
-                                    koodityyppi.nimi['fi'] ||
-                                    koodityyppi.nimi['sv'] ||
-                                    koodityyppi.nimi['en']
-                                );
-                            }
-                            return tyyppi;
-                        })
+                if (row.original.ryhmatyypit.length > 0) {
+                    return row.original.ryhmatyypit
+                        .map((tyyppi: string) => ryhmaTyypitKoodisto.uri2Nimi(tyyppi.slice(0, -2)))
```

Comment:
slice? why? wtf? could be worked around by .... naming. for example dropSuffix(tyyppi). 
bonus: it could be tested easily

Reply:
This name versioning( when the version number actually is there and when it is not) is a bit of a mystery to me and hence the slicing for now. i will move it to a dropsuffix func as suggested. Thanks!